test/PolygonTests.cs

```diff
@@ -501,6 +501,10 @@ public void PointAt()
             var p = new Polygon(new[]{a,b,c,d});
             Assert.Equal(4, p.Segments().Count());
             Assert.Equal(new Vector3(1.0, 1.0), p.PointAt(0.5));
+
+            var r = Polygon.Rectangle(2,2);
+            Assert.Equal(new Vector3(1,1,0), r.PointAt(0.5));
```

Comment:
This seems like it should belong to #121. Do you want to include it here and close that PR?

Reply:
you are right.  I'd rather include it in the other pull request, I'm just all over the place with my forked branches.  I'm inlcuding it there now....not sure how to un-include this change here.